tests/test_ls.py

```diff
@@ -124,19 +124,17 @@ def test_ls_dircache_remove_uncached(
     prefix = f"{repository}/{temp_branch}"
     resource = f"{prefix}/"
 
-    try:
-        listing_pre = fs.ls(resource)
-        fs.rm(listing_pre[0]["name"])
+    listing_pre = fs.ls(resource)
+    fs.rm(listing_pre[0]["name"])
 
-        # List again, bypassing the cache...
-        listing_post = fs.ls(resource, refresh=True)
-        assert len(listing_post) == len(listing_pre) - 1
+    # List again, bypassing the cache...
+    listing_post = fs.ls(resource, refresh=True)
+    assert len(listing_post) == len(listing_pre) - 1
 
-        # ... and through the cache (which should have been updated above)
-        listing_post = fs.ls(resource)
-        assert len(listing_post) == len(listing_pre) - 1
-    finally:
-        client_helpers.reset_branch(fs.client, repository, temp_branch)
+    # ... and through the cache (which should have been updated above)
+    listing_post = fs.ls(resource)
+    assert len(listing_post) == len(listing_pre) - 1
+    client_helpers.reset_branch(fs.client, repository, temp_branch)
```

Comment:
I'm guessing this is also inconsequential like in the other test below, where you deleted the branch reset.

Reply:
Removed in 926fb3be2d1eee23b75dfc9514e8570acff35519.